docs/UserGuide.md

```diff
@@ -473,53 +531,95 @@ is shown in the screenshot above. In the actual application, you can scroll down
 ### 3. Commands - Manpower Management <a name="commands-manpower-management"></a>
 #### 3.1 Adding an employee : `c-add`
 
-Adds an employee to the contact list.
+Adds an employee to the Employee Directory.
 
 Format: `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG] ...`
 
-<div markdown="span" class="alert alert-primary">:bulb:
 
-**Tip:**
-An employee can have any number of tags (including 0)
+* In tCheck, you may add an employee to the Employee Directory. When you add an employee, you need to provide his/her name, phone number, emergency contact, and address.
+* Tag is optional. Tags names should be alphanumeric. White space, characters are not allowed to be used in tags names. 
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:** <br>
+* PHONE_NUMBER and EMERGENCY_CONTACT must be 8-digit number and start with 8 or 9.<br>
 </div>
 
-Examples:
-* `c-add n/John Doe p/98765432 e/81234567 a/Blk 123 ABC Road`.
-* `c-add n/Betsy Crowe e/81234567 p/89007413 a/Blk 120 ABC Road t/Friday t/PartTime`.
+Example: 
 
-#### 3.2 Listing all employees : `c-list`
+Let's say you want to add an employee called John Doe to the Employee Directory. His phone number is 98765432. His emergency contact is 81234567. His address is Blk 123 ABC Road. Assume he is a part-time employee, and he only can work on Friday. So you want to indicate these two characteristics in tCheck. You can assign him two labels, "Friday" and "PartTime" so that you can use find-employees-by-tags features (refer to `c-tag-find` for more details) to find a group of employees by their tag(s).You can follow these instructions:
 
-Shows a list of all employees in the contact list.
+1. Type `c-add n/John Doe p/98765432 e/81234567 a/Blk 123 ABC Road t/Friday t/PartTime` into the _Command Box_.
+1. Press "Enter" to execute.
 
 
-Format: `c-list`
+Outcome:
+1. The _Result Display_ will show that you add a new employee.
+2. All active employees will be listed in the Employee Directory inside the _Main View_.
 
-#### 3.3 Editing a person : `c-edit`
+![AddEmployeeCommandScreenshot](images/AddEmployeeResult.png)
```

Comment:
Picture is missing. Did you remember to add it into this PR and confirm the name?